pyproject.toml

```diff
@@ -57,13 +85,9 @@ norecursedirs = [
   "__pycache__",
 ]
 DJANGO_SETTINGS_MODULE = "settings_test"
-# Ignoring csp deprecation warnings, we have control over the module and
-# currently it warns for child-src which is deprecated in CSPv3 but we're still
-# on CSP 2 while CSP 3 is still in working draft (sept 2018)
```

Comment:
This comment is still useful (but should be moved below)

Reply:
REverted